lib/collection.js

```diff
@@ -310,6 +309,7 @@ const DEPRECATED_FIND_OPTIONS = ['maxScan', 'fields', 'snapshot', 'oplogReplay']
  * @param {boolean} [options.noCursorTimeout] The server normally times out idle cursors after an inactivity period (10 minutes) to prevent excess memory use. Set this option to prevent that.
  * @param {object} [options.collation] Specify collation (MongoDB 3.4 or higher) settings for update operation (see 3.4 documentation for available fields).
  * @param {boolean} [options.allowDiskUse] Enables writing to temporary files on the server.
+ * @param {string|boolean} [options.explain] The verbosity mode for the explain output.
  * @param {ClientSession} [options.session] optional session to use for this operation
```

Comment:
I was not sure the best way to communicate what the valid values of `options.explain` are. We could list the valid strings and how the boolean values will be interpreted, which is what we do for `readPreference`, or we could link to some server documentation, which is what we do for `hint`.

Reply:
I did suggest lower to put the strings in, if we put the strings in VSCode can suggest the strings to devs 🤷  but then it might error if there's a new string later.. Maybe the link is best

Reply:
If we go ahead with using the enum from 4.0 to check that the string passed in is a valid verbosity, then it will error if there's a new verbosity string added later anyway. 

Since we went ahead with doing the valid verbosity check in 4.0, my thought is to go ahead and do that here as well, and list the strings within the type. More work for us later if a new string is added, though.